### PR TITLE
fix(Upload): circle reference

### PR DESCRIPTION
**Pull request by kanweiwei**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#34006 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     fix circle reference between `Upload`  and `Dragger`     |
| 🇨🇳 Chinese |     修复 `Upload` 和 `Dragger` 之间的循环依赖      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
图片

**Comment by afc163** [comment]
ping @kanweiwei

**Comment by afc163** [comment]
ping @kanweiwei

File: components/upload/index.tsx [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+interface UploadInterface extends InternalUploadType {
+  Dragger: typeof Dragger;
+  LIST_IGNORE: string;
+}
+
+const Upload = InternalUpload as UploadInterface;
+
 Upload.Dragger = Dragger;
+
+Upload.LIST_IGNORE = LIST_IGNORE;
```
Comment (afc163) [inline components/upload/index.tsx:17]:
```suggestion
const Upload = InternalUpload as UploadInterface;
Upload.Dragger = Dragger;
Upload.LIST_IGNORE = LIST_IGNORE;
```

**Comment by kanweiwei** [comment]
I'll check it today

**Comment by afc163** [comment]
You can rebase lastest master branch.

**Comment by afc163** [comment]
图片

**Pull request merged**